modules/pam_namespace/pam_namespace.c

```diff
@@ -38,6 +38,7 @@
 #include "pam_inline.h"
 #include "pam_namespace.h"
 #include "argv_parse.h"
+#include <stdint.h>
```

Comment:
Thank you for the PR. It's a shame that this was neither triggered on my system nor in the pipelines. It would be nice if you could move it above the other includes for consistency reasons.

Otherwise definitely fully agree, of course.